src/iqstat.cpp

```diff
@@ -363,3 +241,94 @@ void usage() {
     }
     printReleaseInfo( "iqstat" );
 }
+
+auto show_RuleExec( char *userName, 
+                    const char *ruleName, 
+                    int allFlag, 
+                    bool brief ) -> int
+{
+    genQueryInp_t genQueryInp{};
+    genQueryOut_t *genQueryOut{};
+
+    auto num_cols_selected = (brief ? once_instance.brief_format_len
+                                    : once_instance.long_format_len);
+
+    for (int i = 0; i < num_cols_selected; i++) {
+        addInxIval(&genQueryInp.selectInp,i1a[i],0);
+    }
+
+    if (!allFlag) {
+        snprintf( v1, BIG_STR, "='%s'", userName );
+        addInxVal(&genQueryInp.sqlCondInp,COL_RULE_EXEC_USER_NAME,v1);
+    }
+
+    std::string diagnostic {""};
```

Comment:
The default constructor of a `std::string` results in an empty string.

This line is equivalent to: `std::string diagnostic;`